Cargo.toml

```diff
@@ -75,12 +75,13 @@ bytesize = "1.1.0"
 prettytable = "0.10.0"
 textwrap = "0.16.1"
 
-near-ledger = { version = "0.8.0", optional = true }
+near-crypto = "0.26"
+near-primitives = "0.26"
+near-jsonrpc-client = "0.13"
+near-jsonrpc-primitives = "0.26"
+near-socialdb-client = "0.7"
```

Comment:
I moved it as it's one of the dependencies that we update every time the release happens. So, it makes sense to have it grouped.

Also, I have moved near-ledger to near-gas and near-token with the same logic in mind.